code/__DEFINES/flags.dm

```diff
@@ -42,6 +42,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define ADMIN_SPAWNED_1 (1<<10)
 /// should not get harmed if this gets caught by an explosion?
 #define PREVENT_CONTENTS_EXPLOSION_1 (1<<11)
+/// Should this mob be paintable?
+#define MOB_PAINTABLE_1 (1<<13)
```

Comment:
Please make this a trait instead, we should only use flags where performance is important (either because a lot of things have the flag, or because it's a hot process to check)

Reply:
Whether or not an object is unpaintable or not is set as a flag, and this flag does something extremly similar. Wouldn't changing it into a trait make it inconsistent and confusing to read?

Reply:
flags_1 is reserved for extremely commonly checked flags.

Reply:
We get exactly 24 flags until we have to make a new variable, and make everything more confusing. I want to delay that process.